client/src/ui-components/ShoppingCartComponent/ShoppingCartControlsCSS.css

```diff
@@ -0,0 +1,32 @@
+.cart-inverse-button {
+  border: 2px solid black;
+  background-color: whitesmoke;
+  color: #db9471;
+  width: auto;
+  margin-left: 1%;
+  border-color: #db9471;
+}
+.cart-inverse-button:hover {
+  border: 2px solid black;
+  background-color: #ece6e2;
+  color: #db9471;
+  width: auto;
+  margin-left: 1%;
+  border-color: #db9471;
+}
+.cart-button {
+  border: 2px solid black;
+  background-color: #db9471;
+  width: auto;
+  border-color: #db9471;
+}
+.cart-button:hover {
+  border: 2px solid black;
+  background-color: #d88054;
+  width: auto;
+  border-color: #db9471;
+}
+.modal-title {
+  color: #635f46 !important;
+  font-weight: bold;
+}
```

Comment:
when 'send order 'is clicked but the order is not sent, the button becomes blue.
Possible solution: specifies its behaviour with:  .cart-button:focus